website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+`dc1`, you must set the `datacenter` parameter on the secondary datacenter.
+Otherwise, both datacenters will be named `dc1` and there will be conflicts.
+
+### Consul Servers and Clients
+
+You should explicitly set the  `primary_datacenter` parameter on all servers
+and clients, even though, replication is enabled by default on the primary
+datacenter. Your agent configuration should be similar to the example below.  
+
+```json 
+{ 
+"datacenter": "primary_dc", 
+"primary_datacenter": "primary_dc",
+"acl": { 
+  "enabled": true, 
+  "default_policy": "deny", 
+  "down_policy": "extend-cache",
+  "enable_token_persistence": true 
+  } 
+} 
+```
+
+The `primary_datacenter`
+[parameter](https://www.consul.io/docs/agent/options.html#primary_datacenter)
+sets the primary datacenter to have authority for all ACL information. It is
+important to also set this on the clients, so that the clients can forward API
+requests to the servers. 
+
+Finally, start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+```
+
+Complete this process on all agents. 
+
+## Create the Replication Token for ACL Management
+
+Next create the replication token for managing ACLs. The token should be
+created with the following privileges:
+
+acl = "write" which will allow you to replicate tokens.  operator = "read" for
+replicating proxy-default configuration entries.  service_prefix, policy =
+"read" and intentions = "read" for replicating service-default configuration
+entries, CA, and intention data. 
+
+```hcl 
+acl = "write"
+
+operator = "read"
+
+service_prefix "" { 
+	policy = "read" 
+	intentions = "read" 
+} 
+```
+
+Now that you have the ACL rules defined, create a policy. 
+
+```sh 
+$ consul acl policy create -name replication -rules
+@replication-policy.hcl 
+ID:           240f1d01-6517-78d3-ec32-1d237f92ab58
+Name:         replication 
+Description: Datacenters: 
+Rules: acl = "write"
+
+operator = "read"
+
+service_prefix "" { policy = "read" intentions = "read" } 
+```
+
+Finally, use your newly created policy to create the replication token.
+
+```sh 
+$ consul acl token create description "replication token" -policy-name
+replication 
+AccessorID:   67d55dc1-b667-1835-42ab-64658d64a2ff 
+SecretID:     fc48e84d-3f4d-3646-4b6a-2bff7c4aaffb 
+Description:  replication token 
+Local:        false
+Create Time:  2019-05-09 18:34:23.288392523 +0000 UTC 
+Policies: 
+  240f1d01-6517-78d3-ec32-1d237f92ab58 - replication 
+```
+
+## Enable ACL Replication on the Secondary Datacenter
+
+Once you have configured the primary datacenter and created the replication
+token, you can setup the secondary datacenter. 
+
+### Configure the Servers 
+
+You will need to set the `primary_datacenter` parameter to the name of your
+primary datacenter, `enable_token_replication` to true, and `retry_join_wan` to
+the primary datacenters server's addresses. By configuring the retry join
+parameter for wan gossip, you are also setting up [basic
```

Comment:
As mentioned earlier, perhaps move this up as a precondition of the guide?

Reply:
Done.